core/src/commonMain/kotlin/com/sourcepoint/mobile_core/models/SPError.kt

```diff
@@ -1,5 +1,10 @@
 package com.sourcepoint.mobile_core.models
 
+import kotlin.experimental.ExperimentalObjCName
+import kotlin.native.ObjCName
+
+@OptIn(ExperimentalObjCName::class)
+@ObjCName(swiftName = "CoreSPError")
```

Comment:
We should use typealias in Swift instead. And only resort to `@ObjCName(swiftName)` if there's no other solution.